### PR TITLE
Add script and voters list for Feb 2021 elections

### PR DESCRIPTION
**Pull request by afrittoli**
Add a second script to resolve github user ids, that reduces a bit
the amount of work that needs to be done with the original script.

Add an updated user list for 2021.

Signed-off-by: Andrea Frittoli <andrea.frittoli@gmail.com>

### CODE REVIEW

File: election/github_emails.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/user/bin/env python3
```
Comment (bobcatfish) [inline election/github_emails.py:1]: 😅

File: election/github_emails_2.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+"""get_emails_2.py Extracts github IDs from a CSV and matches them
```
Comment (bobcatfish) [inline election/github_emails_2.py:3]: any reason not to replace the other script?

**Review by bobcatfish — COMMENTED** [review]
mostly just wondering if we could delete the previous script in favor of the new one

File: election/github_emails_2.py (hunk 2 above)
Comment (afrittoli) [reply election/github_emails_2.py:3]: You can run this first on the output of grafana, and then run the other script on the missing list produced by this script.
Reason being that the first script is more likely to give clean results, but also to have many miss.
I should have documented this better? Or maybe aggregated the two steps? I was a bit lazy :P

**Review by iancoffey — COMMENTED** [review]
this lgtm, thanks for making this

**Review by vdemeester — APPROVED** [review]

**Pull request merged**